CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Next
 
+### Fixed
+- Fix a bug where `openai` Python client and `numpy` were required to import any embedder or LLM.
```

Comment:
Shall we have the alpha-releases in the CHANGELOG or we can put everything in the next release?

Reply:
We have the alpha release in the Changelog, right? The only thing is that we'll need to copy all the changes under Next before releasing the final version.